src/models/Call.ts

```diff
@@ -537,3 +552,253 @@ export class JitsiCall extends Call {
         this.setDisconnected();
     };
 }
+
+interface ElementCallMemberContent {
+    "m.expires_ts": number;
+    "m.calls": {
+        "m.call_id": string;
+        "m.devices": {
+            "m.device_id": string;
+        }[];
+    }[];
+}
+
+/**
+ * A group call using MSC3401 and Element Call as a backend.
+ * (somewhat cheekily named)
+ */
+export class ElementCall extends Call {
```

Comment:
Since it is a separate class, how about extracting it into its own file?

Reply:
Unfortunately with the way I've structured this that isn't possible. Call depends on ElementCall, and ElementCall depends on Call, so putting them in separate files would create an import loop.

Reply:
Oh I see. I had such issues while working on the new DM flow. Do we already have a `models` package? That might break the loop. Anyway low priority.